### DISCUSSION TITLE
High Availability / Redundancy

### DISCUSSION THREAD

**Discussion by thomasklosinsky** [mikopbx/Core#100] [Ideas (Идеи)]
Hi,
is there any HA or redundancy feature/module available or planned?

And any possibility to install mikopbx on Ubuntu?

**Comment by boffart** [comment]
Hi
There is no such possibility yet. 
It is possible to back up data via sftp. 
We plan to install mikopbx on debian, but this is not a priority.

Sample demo installation script for v.**2020.1.124**:
https://files.miko.ru/s/p6BM3FoajO7XV66/download

**Reply by thomasklosinsky** [reply]
would it be possible to create a module which handles HA?
which data needs to be synced? only the asterisk config files? what about databases?

**Reply by boffart** [reply]
It is enough to synchronize
/dev/xxx3 on **/cf** type ext4 (rw,relatime,data=ordered)
/dev/xxx1 on **/storage/usbdisk1** type ext4 (rw,relatime,data=ordered)

Excluding the log directory: **/storage/usbdisk1/mikopbx/log/**
